### DISCUSSION TITLE
在w11下 w10磁贴打不开

### DISCUSSION THREAD

**Discussion by postyizhan** [zetaloop/ExplorerPatcher#41] [Q&A] [answered]
https://github.com/zetaloop/ExplorerPatcher/assets/97342038/39c4574a-83bb-40c8-b49d-49a4832e2fbc

尝试过
https://github.com/zetaloop/ExplorerPatcher/discussions/31
https://github.com/zetaloop/ExplorerPatcher/discussions/34

请问我需要提供哪些信息

**Comment by zetaloop** [comment, marked as answer]
你现在在用哪个系统版本？（设置-系统-系统信息-Windows规格-操作系统版本）是内测吗？
这是哪个版本的EP？是预览版吗？
用官方原版（ https://github.com/valinet/ExplorerPatcher ）也没法打开win10开始菜单吗？

**Reply by postyizhan** [reply]
版本 Windows 11 专业版
版本 22H2
安装日期 ‎2023/‎10/‎20
操作系统版本 22621.2428
体验 Windows Feature Experience Pack 1000.22674.1000.0

原版还没有测试

**Reply by postyizhan** [reply]
测试了 原版是可以使用的

**Reply by zetaloop** [reply]
你是否刚刚把开始菜单**显示位置**从“居中”改成“屏幕边缘”了？
我这边试了一下，开始菜单会出现在很下面卡住出不来，但是如果修改一下**显示模式**然后再改回来就好了。
官方版和汉化版好像一样都有这个问题。

**Reply by postyizhan** [reply]
抱歉，高中生不能及时回复消息，已经解决问题，感谢您的帮助
